recipes/libyuv/all/patches/001-cmake.patch

```diff
@@ -0,0 +1,55 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -6,6 +6,9 @@
+ CMAKE_MINIMUM_REQUIRED( VERSION 2.8 )
+ OPTION( TEST "Built unit tests" OFF )
+ 
++include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
++conan_basic_setup()
++
+ SET ( ly_base_dir	${PROJECT_SOURCE_DIR} )
+ SET ( ly_src_dir	${ly_base_dir}/source )
+ SET ( ly_inc_dir	${ly_base_dir}/include )
+@@ -22,13 +25,19 @@
+ 
+ INCLUDE_DIRECTORIES( BEFORE ${ly_inc_dir} )
+ 
++if(MSVC)
++    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
++endif()
++
+ # this creates the static library (.a)
+-ADD_LIBRARY				( ${ly_lib_static} STATIC ${ly_source_files} )
++ADD_LIBRARY				( ${ly_lib_static}  ${ly_source_files} )
++set_property(TARGET ${ly_lib_static} PROPERTY CXX_STANDARD 11)
```

Comment:
Do not force shared library, instead, open an issue request to the upstream. Remove this patch.

Reply:
I do not find any way to report issue for libyuv

Reply:
https://bugs.chromium.org/p/libyuv/issues/list

Reply:
I has report an issue [here](https://bugs.chromium.org/p/libyuv/issues/detail?id=874)

Reply:
According to [this](https://src.fedoraproject.org/rpms/libyuv/blob/master/f/libyuv.spec#_12) comment, it seems that libyuv upstream does not want to fix this.

Reply:
The fedora patches are about only building shared libraries.
Linux distros usually only have the packages available as a shared library.